includes/widgets/star-rating.php

```diff
@@ -209,29 +209,41 @@ protected function register_controls() {
 			]
 		);
 
+		$rtl   = is_rtl();
+		$start = $rtl ? 'right' : 'left';
+		$end   = ! $rtl ? 'right' : 'left';
+
 		$this->add_responsive_control(
 			'align',
 			[
 				'label' => esc_html__( 'Alignment', 'elementor' ),
 				'type' => Controls_Manager::CHOOSE,
 				'options' => [
-					'left' => [
-						'title' => esc_html__( 'Left', 'elementor' ),
-						'icon' => 'eicon-text-align-left',
+					'start' => [
+						'title' => esc_html__( 'Start', 'elementor' ),
+						'icon' => "eicon-text-align-$start",
 					],
 					'center' => [
 						'title' => esc_html__( 'Center', 'elementor' ),
 						'icon' => 'eicon-text-align-center',
 					],
-					'right' => [
-						'title' => esc_html__( 'Right', 'elementor' ),
-						'icon' => 'eicon-text-align-right',
+					'end' => [
```

Comment:
What happens if someone previously selected the `right` alignment and then updates to the new version of the plugin?

Reply:
The `classes_dictionary` and `selectors_dictionary` args are for backwards compatibility.

See: https://developers.elementor.com/docs/editor-controls/selectors-dictionary/